blocks/init/src/Blocks/custom/tab/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+	"blockName": "tab",
+	"title": "Tab",
+	"description" : "Tab block with custom settings.",
+	"category": "eightshift",
+	"icon": {
+		"src": "es-column"
+	},
+	"keywords": [
+		"tab"
+	],
+	"parent": [
+		"eightshift-boilerplate/tabs"
+	],
+	"example": {
+		"innerBlocks": [
+			{
+				"name": "eightshift-boilerplate/card",
+				"attributes": {
+					"imageUrl": "https://loremflickr.com/800/800",
```

Comment:
Another issue that came up with the example is that the editor code heavily relies on using block client IDs, which isn't available using `wp.data.select('core/block-editor')` in the example modal and Storybook.

I was talking to @goranalkovic-infinum about this, who suggested just returning a dummy example, which isn't particularly useful for the Storybook (although it could be a decent solution for content editors)

I'd like some additional input about this. :)

/cc @dingo-d